### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by dvan2**
Hey guys.  I tried to clean up the Navbar using react boot strap: https://react-bootstrap.netlify.app/docs/components/navbar/

Let me know if there's any suggestion.

### CODE REVIEW

File: cs464-group-project/src/components/navigationBar/NavBar.js [hunk 1]
```diff
@@ -1,32 +1,23 @@
-import React from 'react';
-import { Navbar, Nav, NavItem, NavLink, Container } from 'reactstrap';
-import '../../style/NavigationBar.css';
+import React from "react";
+import Container from "react-bootstrap/Container";
```
Comment (txiong002) [inline cs464-group-project/src/components/navigationBar/NavBar.js]: Remove if not using Container

**Review by txiong002 — CHANGES_REQUESTED** [review]
Can you add some contrast to the nav bar and when it hovering, its really hard to see. 
image

**Review by txiong002 — CHANGES_REQUESTED** [review]
Can you center the text when opening the hambuger window. Also can  you make the underline just span the length of text  when in hambuger mode.
image

**Comment by dvan2** [comment]
> Can you center the text when opening the hambuger window. Also can you make the underline just span the length of text when in hambuger mode. image

Here is the new Update: 
image

**Review by txiong002 — APPROVED** [review]
Looks good!

**Pull request merged**